tests/TestCase/ORM/QueryTest.php

```diff
@@ -2993,22 +2993,21 @@ public function testLeftJoinWithNested()
 
         $results = $table
             ->find()
-            ->select(['total_articles' => 'count(articles.id)'])
+            ->select([
+                'authors.id',
+                'total_articles' => 'count(tags.id)'])
```

Comment:
Misleading select alias, I guess. It is the count of articles with that particular tag